pepper-apis/src/main/java/org/broadinstitute/ddp/model/activity/instance/question/TextQuestion.java

```diff
@@ -82,34 +88,12 @@ public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemp
         }
     }
 
-    public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemplateId, List<TextAnswer> answers,
-                        List<Rule<TextAnswer>> validations, TextInputType inputType, SuggestionType suggestionType,
-                        List<String> suggestions, boolean confirmEntry,
-                        Long confirmPromptTemplateId, Long mismatchMessageTemplateId) {
-        this(stableId,
-                promptTemplateId,
-                placeholderTemplateId,
-                false,
-                false,
-                false,
-                null,
-                null,
-                null,
-                answers,
-                validations,
-                inputType,
-                suggestionType,
-                suggestions,
-                confirmEntry,
-                confirmPromptTemplateId,
-                mismatchMessageTemplateId);
-    }
-
-    public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemplateId,
+    public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemplateId, Long confirmPlaceholderTemplateId,
```

Comment:
Does it make sense to pass in `confirmPlaceholderTemplateId` when we're not passing in other properties like confirmEntry, confirmPrompt, etc? I think it'll make more sense to add `confirmPlaceholderTemplateId` to the construct that was deleted above.

Reply:
But in this constructor other confirm-ids also passed:. This version from develop:
```
public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemplateId, List<TextAnswer> answers,
                        List<Rule<TextAnswer>> validations, TextInputType inputType, SuggestionType suggestionType,
                        List<String> suggestions, boolean confirmEntry,
                        Long confirmPromptTemplateId, Long mismatchMessageTemplateId)
```

Reply:
I can return back that 1st constructor. But it is not used anywhere.

Reply:
For this constructor I'm looking at (line 91)
```java
public TextQuestion(String stableId, long promptTemplateId, Long placeholderTemplateId, Long confirmPlaceholderTemplateId,
                    List<TextAnswer> answers, List<Rule<TextAnswer>> validations, TextInputType inputType) {
```
why is confirmPlaceholder there when other properties like confirmEntry, confirmPrompt, etc are not?

Reply:
The constructor you posted from develop is not the one I'm talking about. That one is fine and we should add confirmPlaceholder to it. But this one I'm pointing out shouldn't have confirmPlaceholder -- it doesn't make sense here unless we also pass in other "confirm" properties too.

Reply:
Ah, sorry.. Fixed.